Common_Words_Scraping.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+import time
+from lxml import html
+import requests
+import re
+from bs4 import BeautifulSoup
+import os
+import csv
+from Wiktionary_scraper import *
```

Comment:
not a good practice. Better only import and use it as prefix, or if you find it too long, use an alias. `import <module> as <alias>`. It then ease the reading of the code, since we know where the function come from.